game.js

```diff
@@ -354,7 +354,6 @@ class Game {
             'game_id': this.game_id,
             'move': encodeMove(move)
         }));
-        //this.sendChat(`Test chat message, my move #${move_number}  is: ${move.text}`, move_number, "malkovich");
```

Comment:
why do we remove this line?

Reply:
It seemed like a leftover test that should be.

Reply:
yeah small improvements like this are ok. They do slow down the review process, so separate them if it becomes too big. (but that holds true for any pr)